docs/wetboek.md

```diff
@@ -757,7 +757,10 @@
 
 ### Artikel V-3 Steek- en slagwapens (27 WWM)
 
-1. Strafbaar is een persoon die een steek- of slagwapen voorhanden heeft, bij zich draagt, of opgeslagen heeft in huis of in de laadruimte van zijn of haar vervoersmiddel, waarvan redelijkerwijs kan worden aangenomen dat deze bestemd is om letsel toe te brengen.
+1. Strafbaar is een persoon die een steekwapen voorhanden heeft, bij zich draagt, of opgeslagen heeft in huis of in de laadruimte van zijn of haar vervoersmiddel.
```

Comment:
kofferbak mag wel? je mag normaal ook gewoon transporteren in WWM je mag het niet voorhanden hebben.

Reply:
Ook het vervoeren is verboden. 
![image](https://user-images.githubusercontent.com/71287080/161444596-b7f735c4-a930-4b67-ad1e-b3e284bf4adc.png)
[Artikel 13 Wet wapens en munitie](https://wetten.overheid.nl/jci1.3:c:BWBR0008804&paragraaf=3&artikel=13&z=2019-05-01&g=2019-05-01)

(Categorie I is messen en dergelijke)

Reply:
![image](https://user-images.githubusercontent.com/16556575/161444726-90a693aa-faea-4c7f-9853-7689fa275db9.png)

Reply:
Wat bedoel je hiermee Willem? We hebben hierover met justitie leiding en politieleiding gebrainstormd. De wetgeving wijkt inderdaad enigszins af van het echte leven omdat de situatie in TDA anders is dan IRL.